entries/entry02.md

```diff
@@ -1,7 +1,21 @@
 # Entry 2
```

Comment:
Sources: 1
EDP: 0
Knowledge: 0
Skills: 0
Professionalism: 1

Use the rubric! Show some code!